### DISCUSSION TITLE
umi4联邦组件

### DISCUSSION THREAD

**Discussion by zch-123456** [umijs/umi#12188] [General]
请问一下，如何将exposes下的联邦组件打包到remote.js中的时候不做splitChunk，现在项目里面把打包出来的remote.js放到了别的目录下(只有这一个文件)，我看现在remote.js里面会引入同目录下别的js，放到别的目录下就不行了，有什么解决方法不

**Comment by fz6m** [comment]
umi 4 的产物结构是不能自己修改的，请原封不动的使用整个产物文件夹。

如有问题，需要给一个最小复现。
